### PR TITLE
feat: add topic 5 as AsciiDoc page

### LINKED ISSUE #67: feat: add topic 5 as AsciiDoc page

### PR DESCRIPTION
**Pull request by Dzmitry-Highlander**
<html>
<body>

https://proselyte.net/spring-tutorial-full-version/introduction-into-beans/
--

</body>
</html>

### CODE REVIEW

**Review by KateKoml — DISMISSED** [review]

File: index.adoc [hunk 1]
```diff
@@ -8,5 +8,4 @@
 
 Wiki for 'Example'.
```
Comment (KateKoml) [inline index.adoc]: Wiki for 'Spring'

**Review by KateKoml — DISMISSED** [review]

**Review by SvetaPiven — DISMISSED** [review]
great)

File: index.adoc [hunk 2]
```diff
@@ -9,3 +9,4 @@
 Wiki for 'Spring'.
 
 include::spring/intro.adoc[leveloffset=1]
+include::spring/introduction-into-beans.adoc[leveloffset=1]
```
Comment (rakovets) [inline index.adoc]:
```suggestion
include::spring/intro.adoc[leveloffset=1]

include::spring/introduction-into-beans.adoc[leveloffset=1]
```

File: spring/introduction-into-beans.adoc [hunk 3]
```diff
@@ -0,0 +1,66 @@
+= Введение в Beans
+:imagesdir: ../assets/img/spring
+
+*Beans* – это объекты, которые являются основой приложения и управляются *Spring IoC* контейнером. Эти объекты создаются с помощью конфигурационных метаданных, которые указываются в контейнере:
+
+.XML
```
Comment (rakovets) [inline spring/introduction-into-beans.adoc]:
```suggestion
```

File: spring/introduction-into-beans.adoc [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+* Конфигурационный XML-файл
+* Конфигурация на основе аннотаций
+* Конфигурация на основе Java
+
+Пример XML-файла с различными свойствами:
+
+`xml-config.xml`
+
+image::xml-config.webp[Пример конфигурации]
```
Comment (rakovets) [inline spring/introduction-into-beans.adoc]:
```suggestion
.`xml-config.xml`
image::xml-config.webp[Пример конфигурации]
```

File: spring/introduction-into-beans.adoc [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+Определение *bean* содержит метаданные конфигурации, которые необходимы управляющему контейнеру для получения следующей информации:
+
+* Как создать *bean*
+* Информацию о жизненном цикле бина
+* Зависимости *bean*
+
+В *Spring Framework* существуют такие свойства, определяющие *beans*:
+
+''''
```
Comment (rakovets) [inline spring/introduction-into-beans.adoc]:
```suggestion
```

Comment (rakovets) [reply spring/introduction-into-beans.adoc]: Не патрэбна паўтараць нейкія фішкі дызайну аўтара

File: spring/introduction-into-beans.adoc [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+Определеят метод уничтожения *bean*, который будет использоваться при уничтожении контейнера, содержащего *bean*.
+
+*autowiring mode*
+
+Определяет режим автоматического связывания при внедрении зависимости.
+
+*lazy-initialization mode*
+
+Режим ленивой инициализации даёт контейнеру *IoC* команду создавать экземпляр бина при первом запросе,
+а не при запуске приложения.
```
Comment (rakovets) [inline spring/introduction-into-beans.adoc]:
```suggestion
* *class* +
Этот атрибут является обязательным и указывает конкретный класс Java-приложения, который будет использоваться для создания *bean*.

* *name* +
Определяет области видимости создаваемых объектов.

* *constructor-arg* +
Определяет конструктор, использующийся для внедрения зависимости.

* *properties* +
Определяет свойства внедрения зависимости.

* *initialization method* +
Определяет метод инициализации *bean*.

* *destruction method* +
Определеят метод уничтожения *bean*, который будет использоваться при уничтожении контейнера, содержащего *bean*.

* *autowiring mode* +
Определяет режим автоматического связывания при внедрении зависимости.

* *lazy-initialization mode* +
Режим ленивой инициализации даёт контейнеру *IoC* команду создавать экземпляр бина при первом запросе, а не при запуске приложения.
```

File: spring/introduction-into-beans.adoc [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+
+'''''
```
Comment (rakovets) [inline spring/introduction-into-beans.adoc]:
```suggestion
```

**Review by rakovets — CHANGES_REQUESTED** [review]

**Review by rakovets — APPROVED** [review]

**Comment by rakovets** [comment]
need to resolve merge-conflict

**Review by SvetaPiven — DISMISSED** [review]

**Review by rakovets — DISMISSED** [review]

**Review by KateKoml — DISMISSED** [review]

**Review by KateKoml — APPROVED** [review]

**Review by SvetaPiven — APPROVED** [review]

**Review by rakovets — APPROVED** [review]

**Pull request closed without merging**